erpnext/templates/includes/integrations/authorizenet_checkout.js

```diff
@@ -52,3 +53,14 @@ $('input[data-validation="digit"]')
 	.keypress(function(event) {
 		return (event.charCode !== 8 && event.charCode === 0 || (event.charCode >= 48 && event.charCode <= 57));
 	});
+
+$('#card-number').on('keyup', function () {
+	var val = $(this).val();
```

Comment:
How does this handle backspace? I presume it will keep injecting a space after you delete it with backspace?

You should be trapping it and prevent inserting the space when you see a backspace

ex:
```
$('#card-number').on('keyup', function (e) {
  if ( e.keyCode == 8 ) { 
    return;
  }
```

Haven't tested so I assume that would be happening but I could be wrong.

Reply:
Hey @DeveloperMetal 
The backspace is handled well in this, GIF attached